src/plugins/data/common/search/search_source/search_source.test.ts

```diff
@@ -107,13 +109,7 @@ describe('SearchSource', () => {
   describe('#getFields()', () => {
     test('gets the value for the property', () => {
       searchSource.setField('aggs', { i: 5 });
-      expect(searchSource.getFields()).toMatchInlineSnapshot(`
-        Object {
```

Comment:
I found that several of the inline snapshots in this file was causing syntax highlighting rendering issues in my editor, so I went ahead and simplified the inline snapshots that didn't need to be snapshots, or didn't need to be inline.